docs/advanced-usage/access-control/voms-proxy/index.md

```diff
@@ -0,0 +1,65 @@
+# Voms-proxy
+
+If your workflow needs special permissions that your Virtual Organization entitles you to, the VOMS client can create a proxy certificate for your authentication.
```

Comment:
I wonder if we should explain better what a Virtual Organization is (I personally don't know 🤷‍♂ ) or perhaps link it to some website that explains the concept? Same for VOMS, what does it stand for and so on.

Reply:
That's a good idea, I can add some info on VO, something like..(?)

> VOs (Virtual Organizations) administer users and grant them permissions according to their group, role and capability. At CERN the different experiments such as ATLAS, CMS and LHCb all have their own VO. Being part of a VO can give you access to globally distributed data analysis infrastructure and databases. VOMS (Virtual Organization Membership Service) provide information on the user's relationship with their VO.

Reply:
It looks good to me! let's see what @diegodelemos @tiborsimko think